src/mono/mono.proj

```diff
@@ -70,6 +71,10 @@
     <_MonoCMakeArgs Include="-DENABLE_WERROR=1"/>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(ContinuousIntegrationBuild)' == 'true'">
```

Comment:
Should this rather be a new Mono subset (https://github.com/dotnet/runtime/blob/main/eng/Subsets.props#L94) instead of this?

Reply:
That would be nice but I'm not sure how hard it would be since it's built as part of the mono runtime right now, we'd need to separate that out.

We could set `/p:MonoMsCorDbi=true` in our CI build scripts instead of relying on the `ContinuousIntegrationBuild` propertz here in the meantime.